requirements.txt

```
@@ -2,7 +2,8 @@
 #     stable releases. We do this because `pip freeze` will not capture options (e.g. --extra-index-url) or
 #     platform-specific requirements (e.g. sys.platform == 'win32')
 
-dipy
+# Avoid method=restore bug: https://github.com/spinalcordtoolbox/spinalcordtoolbox/issues/4209#issuecomment-1739644328
+dipy<1.6.0,>1.7.0
```

Whoops! This is not valid pip version range syntax. (pip treats this as an `AND` rather than an `OR`, so there's no way to satisfy both conditions.)

```suggestion
dipy!=1.6.*,!=1.7.*